.github/workflows/on_pull_request_target.yaml

```diff
@@ -0,0 +1,22 @@
+on:
+  pull_request_target:
+    types: [opened, ready_for_review]
+
+jobs:
+  auto_merge_and_request_review:
+    runs-on: ubuntu-22.04
```

Comment:
To avoid work triggered by draft PR open, please add following.
if: ${{ github.event.pull_request.draft == false }}